backend/src/main/java/com/soen341/instagram/exception/like/NoLikeException.java

```diff
@@ -0,0 +1,15 @@
+package com.soen341.instagram.exception.like;
+
+public class NoLikeException extends RuntimeException{
+
+	public NoLikeException() {
+		super("There is no like associated with this account on this post.");
+		// TODO Auto-generated constructor stub
```

Comment:
probably should remove this auto-generated comment (the one in the other constructor as well)

Reply:
Has this been removed? No update was shown